src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machine = {
+            new Truck(),
+            new Bulldozer(),
+            new Excavator()
+        };
 
+        for (Machine machines : machine) {
```

Comment:
The variable name in the enhanced for-loop should be singular, as it represents a single element from the array. Consider renaming `machines` to `machine`.